docs/NooBaaNonContainerized/ConfigFileCustomizations.md

```diff
@@ -478,6 +478,34 @@ Warning: After setting this configuration, NooBaa will skip schema validations a
     3. systemctl restart noobaa
     ```
 
+### 33. Notification space monitor frequency flag -
+* <u>Key</u>: `NOTIFICATION_REQ_PER_SPACE_CHECK`
+* <u>Type</u>: Positive integer
```

Comment:
I think it was fine "Number" (I looked at existing example like `GPFS_DOWN_DELAY` and `NSFS_DIR_CACHE_MAX_TOTAL_SIZE`), and mention that it is a positive integer.